<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessBar_Widget</class>
 <widget class="QWidget" name="ProcessBar_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:&quot;Times New Roman&quot;;
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="frame_thumPic">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_thumbPic">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_process">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_videoTitle">
             <property name="text">
              <string>视频名称</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_duration">
             <property name="text">
              <string>时长</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_method">
             <property name="text">
              <string>解密方法</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_doneSize">
             <property name="text">
              <string>已下载</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_speed">
             <property name="text">
              <string>下载速度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="server_id">
             <property name="text">
              <string>server_id</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="styleSheet">
            <string notr="true">QProgressBar{
        border: none;
        color: black;
        text-align: center;
        background: rgb(238, 238, 240);
}
QProgressBar::chunk {
        border: none;
        background: rgb(255, 177, 132);
}</string>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_bit">
           <property name="text">
            <string>比特率</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
